CMakeLists.txt

```diff
@@ -231,3 +232,40 @@ endif()
 if (BUILD_TESTING)
         add_subdirectory(test)
 endif()
+
+################################################################################
+# Packaging
+################################################################################
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
+if(NOT BUILD_SHARED_LIBS)
+        set(CPACK_PACKAGE_NAME "${CPACK_PACKAGE_NAME}-static")
+        if(CMAKE_POSITION_INDEPENDENT_CODE)
+                set(CPACK_PACKAGE_NAME "${CPACK_PACKAGE_NAME}-fpic")
```

Comment:
I wouldm't include `fPIC` in the package name.   Any library for Debian ought to be built with fPIC.